store/src/cli/mod.rs

```diff
@@ -16,4 +20,74 @@ pub struct Cli {
 #[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Subcommand)]
 pub enum Command {
     Serve,
+
+    #[command(subcommand)]
+    Query(Query),
+}
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Subcommand)]
+pub enum Query {
```

Comment:
Should we add some simple comments here so that we get basic hints in the CLI about what the commands do? Probably fine to limit this to top-level commands (i.e., `serve` and `query`).